src/core/identifiers/identifiers.ts

```diff
@@ -1,8 +1,9 @@
 import { BigNumber } from "ethers";
-
 import { isString } from "../utilities/validation";
 import { HexString } from "../../types/Strings";
 
+const JUST_HEX_REGEX = /dsnp:\/\/0x0*|0x0*/gi;
```

Comment:
how about we rename this to `HEX_PREFIX_REGEX` or something else more descriptive?

Reply:
also, it might be a good idea to add a `^` to the beginning to make sure it's at the start of the string